packages/next/next-server/server/config.ts

```diff
@@ -208,6 +212,47 @@ function assignDefaults(userConfig: { [key: string]: any }) {
     }
   }
 
+  if (result?.images) {
+    const { images } = result
+    if (typeof images !== 'object') {
+      throw new Error(
+        `Specified images should be an object received ${typeof images}`
+      )
+    }
```

Comment:
```suggestion
    if (!images || typeof images !== 'object') {
      throw new Error(
        `Specified images should be a valid object but received ${!images ? images || typeof images}`
      )
    }
```
Should we handle `null`/`undefined` here too?

Reply:
I'm following the pattern from i18n here: https://github.com/vercel/next.js/blob/6ec36595e3687afd64b6aa43ca0ea2eb2d8380dc/packages/next/next-server/server/config.ts#L211-L217